test/trivial/deitz/expressions/all_precedence_levels_in_a_single_expression.chpl

```diff
@@ -1,5 +1,5 @@
 class D { var x = 1; }
-class C: D { var y = 2; }
+class C: D { var y = 2; type t = int; }
```

Comment:
Was this added because of lines 5 and 6?

Reply:
I'm assuming that's because removing the `()` in line 6 didn't go well?  Since so much of the rest of the file doesn't involve generic fields, it seems a little out of place for this test.

But I'm not sure it's very important - I'm mostly pointing it out because it made me stop and think

Reply:
The reason it ended up this way is that, at a glance, it looked to me like all of the other lines were repeating the same pattern. I've changed it to modify lines 5 and 6 instead.